ui/elements/IconGroup/IconGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { StyledIconGroup } from './IconGroup.styles';
+import { IconGroupProps } from './types';
+import Icon from '../Icon/Icon';
+
+/**
+ * IconGroup is a component that groups multiple Icon components together.
+ */
+const IconGroup: React.FC<IconGroupProps> = ({
+    icons,
+    children,
+    className,
+    size,
+    color = 'grey',
```

Comment:
lets prefer using a color token from the palette of existing text-color CSS variables found [here](https://grauity.newtonschool.co/?path=/story/atoms-colors--text)

for example: var(--text-emphasis-primary-default, #16191D)

```suggestion
    color = 'var(--text-emphasis-primary-default, #16191D)',
```